app/views/users/show.html.erb

```diff
@@ -2,22 +2,46 @@
 <html>
 <head>
   <title>User Profile</title>
+  <!-- Include Bootstrap CSS -->
+  <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
 </head>
 <body>
   <h1>User Profile</h1>
   
- <% if @user %>
-  <p><strong>Username:</strong> <%= @user.name %></p>
-  <% if @user.photo.present? %>
-  <p><strong>Photo:</strong> <%= image_tag @user.photo, alt: 'User Photo' %></p>
-<% else %>
-  <p><strong>Photo:</strong> No photo available</p>
-<% end %>
-      <p><strong>Bio:</strong> <%= @user.bio %></p>
-<% else %>
-  <p>User not found</p> <!-- Display an error message -->
-<% end %>
-
+  <div class="container mt-4">
+    <!-- User details go here -->
+    <div class="row">
+      <div class="col-md-2 card">
+        <% if @user.photo.present? && @user.photo =~ URI::DEFAULT_PARSER.regexp[:ABS_URI] %>
```

Comment:
- Your design doesn't follow the given wireframe design. You however, fix the bug faced by the previous reviewer.
Try making the image the same height as the card on the right for user details and show the number of posts on the right lower corner.

#### Check your design
![Screenshot from 2023-10-05 18-08-33](https://github.com/Kidd254/BlogApp/assets/80149314/c2c87576-ec11-4df6-b62c-736a9841efa6)